### PR TITLE
added two pictures with dog on them beside the dalmatin

### PR DESCRIPTION
**Pull request by Oleksii-Poltavets**
Hi, I've added two pictures with dogs on them besides the Dalmatian as a comment in the issue asked to do. I also gave the image attribute height='100' to make them all be one height, my prettier automatically changed some spaces in the code and in most cases fixed the position of tags let me know if it's ok.

image

### CODE REVIEW

**Comment by nbkhope** [comment]
1) You should have created a feature branch first before working on the feature. Then commit to that feature branch and make a pull request using that branch.

```
git checkout -b add-more-dog-pictures
# ... do your work ...
git push origin add-more-dog-pictures
```

I'll let it pass for this pull request (PR).

2) Regarding the automatic code formatter. Put yourself in the shoes of a code reviewer, you see a lot of diff noise https://github.com/nbktechworld/full-stack-web-dev/pull/61/files. It's going to take longer to review. 

From the point of view of focused pull requests relevant to the issue, you should try to change code that is only relevant to your task. Avoid touching code that is unrelated. Refrain and resist the urge from performing cosmetic changes just because you think it is pretty (even if it wasnt "you" and was performed with automatic format on save). The codebase should remains as is. 

So I'd recommend you not use code formatter please. Only touch the code that is relevant to your issue.

File: src/index.html [hunk 1]
```diff
@@ -1,53 +1,68 @@
...
-    <button type="button" onclick="nextImage()">Next Image</button>
-    <button type="button" onclick="bark()">🐶🤚Pat the dog</button>
-    <audio class="js-audio-player" src="https://github.com/nbktechworld/introduction-to-html/raw/abcdac58651ba724c40665c82dd55f171534115f/document-two-sections/barking.mp3"></audio>
-    <ol>
-      <li>first point</li>
-      <li>second point</li>
-      <li>third point</li>
-    </ol>
-    <video src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4" controls width="300"></video>
+      <p id="paragraph1">Lorem ipsum <strong>dolor</strong> sit <em>amet</em>, <b>consectetur</b> adipiscing
```
Comment (nbkhope) [inline src/index.html:24]: Do you mind reverting these changes not related to your issue?

Comment (Oleksii-Poltavets) [reply src/index.html:24]: Hey, thanks for the review, sure I'll fix it right away

**Review by nbkhope — APPROVED** [review]

**Comment by nbkhope** [comment]
@Oleksii-Poltavets thanks for addressing the feedback. Looks good to me. This was also an exercise of having someone commenting on your pull request. Please don't take it personal 🙂

**Comment by nbkhope** [comment]
@Oleksii-Poltavets I don't know if you're on the Discord server, so I'm messaging here:

feel free to ask to work on other issues if you'd like and I'll assign you 👍

**Comment by Oleksii-Poltavets** [comment]
Hey, I'm not in discord yet, thanks I'll ask permission if I have time on
weekends, thanks 👍

**Pull request merged**